src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -72,6 +72,19 @@ def __init__(
             new_config["steps_offset"] = 1
             scheduler._internal_dict = FrozenDict(new_config)
 
+        if hasattr(scheduler.config, "clip_sample") and scheduler.config.clip_sample is True:
```

Comment:
As soon as the 0.7.0 is out, we will add `clip_sample` to https://huggingface.co/runwayml/stable-diffusion-v1-5 so that the default scheduler config is 1-to-1 compatible with all other schedulers